projects/zeek/build.sh

```diff
@@ -74,3 +74,10 @@ for f in ${fuzzers}; do
 
     fuzzer_count=$((fuzzer_count + 1))
 done
+
+if [ "${SANITIZER}" = "coverage" ]; then
+  # Normally, base-builder/compile copies sources for use in coverage reports,
+  # but its use of `cp -rL` omits the "zeek -> ." symlink used by #includes,
+  # causing the coverage build to fail.
+  cp -r $SRC $OUT
```

Comment:
Sorry, I should’ve pointed it out in my previous comment. Let’s copy only your project dir here, not the whole $SRC